testsuite/tests/typing-external/pr11392.ml

```diff
@@ -16,9 +16,9 @@ type 'self nat = Z | S of 'self
 external cast : int -> 'self nat as 'self = "%identity"
 ;;
 [%%expect{|
-Line 1, characters 16-41:
+Line 1, characters 37-41:
 1 | external cast : int -> 'self nat as 'self = "%identity"
-                    ^^^^^^^^^^^^^^^^^^^^^^^^^
+                                         ^^^^
```

Comment:
I find it weird that the location does not span the quote but this is the same as in let-bindings. 

I propose to change the location to include the quote. The quote is not included in the AST but is logically part of the ident as suggested by this error message:

```
1 | fun (x : 'a t as 'a) -> ();;
                      ^
Error: This alias is bound to type "'a t" but is used as an instance of type "'a"
       The type variable "'a" occurs inside "'a t"
```

Reply:
I tend to agree that it would be better to include the quote in the location.